resources/views/discussions/show.blade.php

```diff
@@ -32,6 +32,8 @@
 </div>
 
 <div class="space-y-5">
+    <x-search :search="$search"></x-search>
```

Comment:
Can we do this instead or am I wrong in assuming that this is a thing?
```suggestion
    <x-search :search="$search" />
```

Reply:
yes, will update